src/GraphQL/Resolver/DomainContentMutationResolver.php

```diff
@@ -277,20 +277,13 @@ static function (FieldDefinition $fieldDefinition) use ($fieldDefId) {
                 }
             )->first();
 
-            foreach ($fieldErrors['eng-GB'] as $fieldError) {
-                $translatableMessage = $fieldError->getTranslatableMessage();
-                if ($translatableMessage instanceof API\Values\Translation\Plural) {
-                    if (count($fieldError['values']) === 1) {
-                        $message = $translatableMessage->singular;
-                    } else {
-                        $message = $translatableMessage->plural;
-                    }
-                } else {
-                    $message = $translatableMessage->message;
-                }
+            $fieldErrorsList = $fieldErrors['eng-GB'] ?? reset($fieldErrors);
+            $fieldErrorsList = is_array($fieldErrorsList) ? $fieldErrorsList : [$fieldErrorsList];
```

Comment:
Your patch is correct, however the root cause of the issue is hard-coded `eng-GB` language code, which shouldn't happen here :disappointed: 

Can we change `renderFieldValidationErrors`  implementation to:

```php
    private function renderFieldValidationErrors(RepositoryExceptions\ContentFieldValidationException $e, API\Values\ContentType\ContentType $contentType)
    {
        $errors = [];
        foreach ($e->getFieldErrors() as $fieldDefId => $fieldErrorsByLanguage) {
            $fieldDefinition = $contentType->getFieldDefinitions()->filter(
                static function (FieldDefinition $fieldDefinition) use ($fieldDefId) {
                    return $fieldDefinition->id === $fieldDefId;
                }
            )->first();

            foreach ($fieldErrorsByLanguage as $fieldErrors) {
                foreach ($fieldErrors as $fieldError) {
                    // depending on error instance, values injected in Plural::__toString or Message::__toString
                    $errors[] = sprintf("Field '%s' failed validation: %s",
                        $fieldDefinition->identifier,
                        (string)$fieldError->getTranslatableMessage()
                    );
                }
            }
        }

        return $errors;
    } 
```

// cc @ezsystems/php-dev-team @bdunogier

Reply:
hm, I rechecked the flow again and I think we don't need to support array in `$fieldErrors` at all.
`$e->getFieldErrors()` should return this structure:
https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/Base/Exceptions/ContentFieldValidationException.php#L26

What do you think about variant I just pushed?